### DISCUSSION TITLE
Console commands

### DISCUSSION THREAD

**Discussion by Kruk79** [bubuntux/nordlynx#43] [Q&A] [answered]
With NordVpn I can use commands such as nordvpn status or nordvpn settings.

Is there something similar for nordlynx? nordlynx status or nordlynx settings does not work

How to check if nordlynx is actually working correctly? I can see in the log status connected. Is this sufficient?

**Comment by ginodesilva** [comment, marked as answer]
Nordlynx uses wireguard, so you can check with the wireguard commands. Such as "wg"

```
root@f1cc2202d46f:/# wg
interface: wg0
  public key: XXXX
  private key: (hidden)
  listening port: 51820

peer: XXXXX
  endpoint: 213.232.87.65:51820
  allowed ips: 0.0.0.0/1, 128.0.0.0/1
  latest handshake: 18 seconds ago
  transfer: 4.71 GiB received, 485.02 MiB sent
  persistent keepalive: every 25 seconds
root@f1cc2202d46f:/# 
```

Also, you can simply use `curl ifconfig.me` and check the IP address you are using to traverse the Internet. Or curl the nordvpn api: `curl https://api.nordvpn.com/vpn/check/full`

I have a healthcheck in my compose which checks the status: 

```
healthcheck:
   test: ["CMD-SHELL", "curl https://api.nordvpn.com/vpn/check/full | grep '\"status\":\"Protected\"' && exit 0 || exit 1"]
```

Hope this helps.

**Comment by Kruk79** [comment]
Many thanks, this is what I was looking for.
